### DISCUSSION TITLE
Edit JSON Objects in SQL

### DISCUSSION THREAD

**Discussion by vwake7** [stackql/stackql#401] [Q&A] [answered]
@jeffreyaven My use case is I have to edit (delete and replace values) JSON object (usually 'properties' Object from Azure Provider). Originally I was planning to do this in stackql SQL but I cannot find the related JSON functions ( json_replace() and  jsonb_remove()) in stackql.  Currently I am planning to use Sqlite JSON functions. I assume I can also use some Python functions. But is this the suggested approach or am I missing something?

**Comment by jeffreyaven** [comment, marked as answer]
hey @vwake7 we do support `json_remove` and `json_replace`, they just aren't documented, give them a try

**Reply by vwake7** [reply]
oh, ok, sure thanks.

**Reply by jeffreyaven** [reply]
ill raise an issue to get these documented

**Comment by jeffreyaven** [comment]
hi @vwake7 have documented several functions including [`JSON_INSERT`](https://stackql.io/docs/language-spec/functions/json/json_insert) and [`JSON_REPLACE`](https://stackql.io/docs/language-spec/functions/json/json_replace), have a look, we are always looking for goo real world examples if you wanted to raise a PR at [stackql.io](https://github.com/stackql/stackql.io)

**Comment by vwake7** [comment]
Sure, I will, More than happy to be involved in this project.
